.github/workflows/deploy-preprod.yml

```diff
@@ -6,7 +6,12 @@ on:
       docker_image:
         type: string
         required: true
+
   workflow_dispatch:
+    inputs:
+      docker_image:
+        required: true
```

Comment:
Can we remove the `required: true`, at least from the pre-prod workflow? That's not required today (a docker build will run if required).

Reply:
Removed from preprod